analyses/fusion_filtering/run_fusion_merged.sh

```diff
@@ -27,39 +27,48 @@ results_path="results/"
 
 
 # fusion files before and after standardization
-arriba_file="${data_path}/pbta-fusion-arriba.tsv.gz"
-starfusion_file="${data_path}/pbta-fusion-starfusion.tsv.gz"
+arriba_file="${data_path}/fusion-arriba.tsv.gz"
+starfusion_file="${data_path}/fusion-starfusion.tsv.gz"
 standard_arriba_file="${scratch_path}/arriba.tsv"
 standard_starfusion_file="${scratch_path}/starfusion.tsv"
 
 # general filtering parameters
-artifact_filter="GTEx_Recurrent|DGD_PARALOGS|Normal|BodyMap"
+artifact_filter="GTEx|HGNC_GENEFAM|DGD_PARALOGS|Normal|BodyMap"
 reading_frame_filter="in-frame|frameshift|other"
 spanningFragCountFilter=100
 
 
 # relevant gene expression files
-polya_expression_file="${data_path}/pbta-gene-expression-rsem-fpkm.polya.rds"
-stranded_expression_file="${data_path}/pbta-gene-expression-rsem-fpkm.stranded.rds"
+rna_expression_file="${data_path}/gene-expression-rsem-tpm-collapsed.rds"
 
 # reference expression file
-normal_expression_file="${references_path}/Brain_FPKM_hg38_matrix.txt.zip"
+normal_expression_file="${references_path}/gtex_adrenal_gland_TPM_hg38.rds"
```

Comment:
do you want to remove this since you have the name hardcoded in the `00` script?

Reply:
Good call - I modified it to be not hardcoded since this will be used in 03 script as well.